### PR TITLE
improve: update desktop spacing to align with the designs

### PR DESCRIPTION
**Pull request by james-a-morris**
This PR updates the UI to align with the Figma designs.

### CODE REVIEW

File: components/Hero/Hero.tsx [hunk 1]
```diff
@@ -114,27 +114,33 @@ const InnerWrapper = styled.div`
...
-    font: var(--header-sm);
-    margin-bottom: 12px;
-  }
 `;
 
 const Subheader = styled(motion.h2)`
   z-index: 1;
   margin-inline: auto;
-  max-width: max(562px, 80%);
+  max-width: min(562px, 80%);
```
Comment (ryanwolhuter) [inline components/Hero/Hero.tsx:143]: woops! totally my bad on this one lol. great work fixing it

File: components/HowItWorks/HowItWorks.tsx [hunk 2]
```diff
@@ -60,18 +61,15 @@ export function HowItWorks() {
...
-          <Header>The Optimistic Oracle verifies data in stages</Header>
-        </HeaderWrapper>
-        {steps.map(({ header, text, subText, animationData }, index) => (
+        <SectionHeader title="How it works" header="The Optimistic Oracle verifies data in stages" />
+        {steps.map(({ header, text, subText, animationData }, index, arr) => (
           <StepWrapper key={header}>
             <StepNumberWrapper>
-              <StepNumber>0{index + 1}</StepNumber>
-              <StepLine />
+              <StepNumber invert={index % 2 === 1}>0{index + 1}</StepNumber>
```
Comment (ryanwolhuter) [inline components/HowItWorks/HowItWorks.tsx:68]: this looks good but did we give up on the idea of animating the numbers / lines?

File: components/SectionHeader/SectionHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    | string
+    | {
+        text: string;
+        redSuffix?: string;
+      };
+  header: string | JSX.Element;
+  constrainWidth?: boolean;
+};
+
+export function SectionHeader({ title, header, constrainWidth }: SectionHeaderProps) {
```
Comment (ryanwolhuter) [inline components/SectionHeader/SectionHeader.tsx:53]: great refactor

**Review by ryanwolhuter — APPROVED** [review]
fantastic work well done

File: components/VoteParticipation/VoteParticipation.tsx [hunk 4]
```diff
@@ -184,6 +169,10 @@ const ActivityText = styled.p`
   }
 `;
 
+const LottieAnimation = styled(Lottie)`
+  /* scale: 1.5; */
```
Comment (daywiss) [inline components/VoteParticipation/VoteParticipation.tsx:173]: can probably remove the comments

**Review by daywiss — APPROVED** [review]

**Pull request merged**